cupy/cuda/compiler.py

```diff
@@ -368,8 +380,10 @@ def compile_using_nvcc(source, options=(), arch=None,
 
 def _preprocess(source, options, arch, backend):
     if backend == 'nvrtc':
-        options += ('-arch=compute_{}'.format(arch),)
-
+        if not runtime.is_hip and _cuda_hip_version >= 11010:
```

Comment:
On HIP we don't call this function (there are `_preprocess_{hiprtc,hipcc}`) so we can simplify a bit:
```suggestion
        if _cuda_hip_version >= 11010:
```

Reply:
This seems not resolved?

Reply:
oops :)